tests/draft2019-09/optional/format/json-pointer.json

```diff
@@ -139,14 +139,14 @@
                 "valid": true
             },
             {
-                "description": "valid JSON-pointer (URI Fragment Identifier) #1",
+                "description": "not a valid JSON-pointer (URI Fragment Identifier) #1",
                 "data": "#",
-                "valid": true
+                "valid": false
             },
```

Comment:
This is an invalid change. `#` is not a json pointer; it's a URI fragment. That section of the RFC is describing how JSON pointers can be used in URIs; that doesn't make the entire string a JSON pointer itself.

Plus this doesn't have anything to do with the rest of the PR.

Reply:
I've already removed all of these.  I did almost immediately.  I think you're looking at an old commit.